packages/cli/src/commands/install/install.ts

```diff
@@ -4,16 +4,17 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  *
- * @flow
  */
-
-import type {ConfigT} from 'types';
+/* eslint-disable import/namespace, import/default */
```

Comment:
Please don't do this for entire file, just where it's needed. We can remove these inline later when cleaning up

Reply:
still there, please see my message above

Reply:
Removing it won't pass the test and the file `link/unlink` is not typed which is causing `import/namespace` typescript error

Reply:
All I ask for is to disable only the lines that need it. It's easy to forget about global disables, that's why we avoid them.

Reply:
+1